resources/lang/en-US/admin/accessories/message.php

```diff
@@ -26,7 +26,11 @@
         'error'   		=> 'Accessory was not checked out, please try again',
         'success' 		=> 'Accessory checked out successfully.',
         'unavailable'   => 'Accessory is not available for checkout. Check quantity available',
-        'user_does_not_exist' => 'That user is invalid. Please try again.'
+        'user_does_not_exist' => 'That user is invalid. Please try again.',
+         'checkout_qty' => array(
```

Comment:
Inconsequential style point - you can use `[` and `]` here instead of the older `array(` syntax if you wanted here.